scripts/qa_crowbarsetup.sh

```diff
@@ -3698,6 +3698,9 @@ function oncontroller_testsetup
 {
     . .openrc
     oncontroller_prepare_functional_tests
+    if [[ $hacloud = 1 ]] ; then
+        crm_mon --failcounts -1 | grep "fail-count=" && complain 34 "Cluster resources' failures detected"
```

Comment:
The goal is to start low. Having a few errors that are definitely errors caught is much better than having nothing checked and fail counts exceeded already after the initial deployment. 

If you look at any ha log for cloud7, you see failures due to chef trying to instantiate resources that are not yet installed from packages at that point. Which is a trivial error and that should not creep in anymore.